package.json

```diff
@@ -8,13 +8,12 @@
     "build:library": "react-scripts build",
     "build:storybook": "build-storybook",
     "verify": "yarn build:library && yarn build:storybook",
-    "prepublish": "yarn build:library"
+    "prepublish": "yarn build:library",
+    "start": "react-scripts start",
+    "build": "yarn verify"
```

Comment:
this seems kind of crappy. The heroku `create-react-app` buildpack looks for a `build` script. So, there it is. For heroku technically we don't need to build the library.